iphone/Classes/TiUILabel.m

```diff
@@ -436,11 +436,6 @@ -(void)setColor_:(id)color
 -(void)setEllipsize_:(id)value
 {
 	ENSURE_SINGLE_ARG(value, NSNumber);
-	//for bool case and parity with android
-	if ([TiUtils intValue:value] == 1) {
-		[[self label] setLineBreakMode:NSLineBreakByTruncatingTail];
-		return;
-	}
```

Comment:
Cannot remove the old implementation, we need to throw at least a deprecation warning for `boolean` inputs.